source/SIL.AppBuilder.Portal.Frontend/src/ui/components/sidebar/navigation.tsx

```diff
@@ -40,65 +32,61 @@ const MenuItem = ({ onClick, className, ...otherProps }: MenuItem) => {
   );
 };
 
-class Navigation extends React.Component<IProps & i18nProps> {
-  render() {
-    const currentOrganizationId = getCurrentOrganizationId();
-    const hasSelectedOrg = currentOrganizationId && currentOrganizationId.length > 0;
-    const allOrgsSelected = '' === currentOrganizationId;
+export default function Navigation({ closeSidebar }: IProps) {
+  const { t } = useTranslations();
+  const { userTasks } = useUserTasksForCurrentUser();
+  const currentOrganizationId = getCurrentOrganizationId();
+  const hasSelectedOrg = currentOrganizationId && currentOrganizationId.length > 0;
+  const allOrgsSelected = '' === currentOrganizationId;
 
-    const { t, closeSidebar } = this.props;
+  return (
+    <Menu className='m-t-none no-borders' pointing secondary vertical>
+      <MenuItem content={t('sidebar.myTasks', { count: userTasks.length })} to='/tasks' onClick={closeSidebar} />
```

Comment:
changed from the 'name' property because 'name' does a bunch of formatting / stripping of characters...........